python/tests/platform/helper.py

```diff
@@ -63,8 +63,8 @@ def get(path: str, **kw) -> requests.Response:
     return http_request("GET", path, **kw)
 
 
-def get_pipeline(name: str):
-    return get(f"{API_PREFIX}/pipelines/{name}")
+def get_pipeline(name: str, selector: str) -> requests.Response:
```

Comment:
I wondered why you didnt give selector a default of "status"

Reply:
Either default can backfire. I wanted to force the caller to make a choice. If this proves to be an issue, we can add a default.